draft-lundberg-cose-two-party-signing-algs.md

```diff
@@ -405,19 +427,19 @@ This section registers the following values in the IANA "COSE Key Types" registr
   - Value: TBD (Requested assignment -1)
   - Description: Reference to a key pair of key type "OKP"
   - Capabilities: \[kty(-1), crv\]
-  - Reference: {{cose-key-refs}} of this document
+  - Reference: {{cose-key-refs}} of this specification
 
 - Name: Ref-EC2
   - Value: TBD (Requested assignment -2)
   - Description: Reference to a key pair of key type "EC2"
   - Capabilities: \[kty(-1), crv\]
-  - Reference: {{cose-key-refs}} of this document
+  - Reference: {{cose-key-refs}} of this specification
 
-- Name: Ref-ML-DSA
-  - Value: TBD
-  - Description: Reference to a key pair of key type "ML-DSA"
+- Name: Ref-AKP
+  - Value: TBD (Requested assignment -3)
```

Comment:
The pattern I've been toying with for the `Ref-` values is to use the negative of the corresponding key type value, which would be -7 in this case. But I also guess it would be unwise to (even if unofficially) commit half the value space to that purpose. What do you think?

Reply:
Updated to match the pattern in https://github.com/YubicoLabs/cose-two-party-signing-algs-rfc/pull/5/commits/670a57504fbd1839096718350edc8423935f8337.

That's a fine pattern to follow.  I agree with you that we couldn't get away with officially reserving half the space.